### PR TITLE
Saas 14 related config translate lpe

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/views/form_common.js [hunk 1]
```diff
@@ -791,8 +791,22 @@ var AbstractField = FormWidget.extend(FieldInterface, {
...
+            if (self.field.related) {
+                var fieldRelation = self.field.related[0];
+                model.name = self.field_manager.fields[fieldRelation].field.relation;
+                model.id = self.view.datarecord[fieldRelation];
+            }
+            return model;
+        }
+
+        var model = determineModel();
+
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/form_common.js]: func name is weird, but a function is actually not necessary:
```
var model = this.view.model;
var resID = this.view.datarecord.id;
if (this.field.related) {
    var fieldRelation = this.field.related[0];
    model = this.field_manager.fields[fieldRelation].field.relation;
    resID = this.view.datarecord[fieldRelation];
}

var trans = new data.DataSet(this, 'ir.translation');
return trans.call_button('translate_fields', [model, resID, this.name, this.view.dataset.get_context()]).done(function(r) {
    self.do_action(r);
});
```

**Comment by kebeclibre** [comment]
@aab-odoo 
Yes, the issue is in v11.0 as well.
I don't like the commit on res.config either but in saas-14 (I did not check for v11 yet, though I'm pessimistic) there is no way to determine if a model is transient or not (given that only those are just used with create to set up values somewhere else)

File: addons/web/static/src/js/views/form_common.js (hunk 1 above)
Comment (kebeclibre) [reply addons/web/static/src/js/views/form_common.js]: @aab-odoo 
Indeed

**Comment by aab-odoo** [comment]
@ged-odoo any idea?

**Comment by aab-odoo** [comment]
@kebeclibre I still believe that we should add a view attribute (in master) to handle this case, and keep the webclient generic

**Comment by rdeodoo** [comment]
@aab-odoo can we close?

**Comment by aab-odoo** [comment]
@rdeodoo I don't know, we should ask @kebeclibre

**Comment by rdeodoo** [comment]
It has been over a year, this has just been forgeted IMO (as many others..), I'll close it as no response.
Reopen if needed.

**Pull request closed without merging**